packages/langchain/src/openai/README.md

```diff
@@ -46,30 +44,48 @@ import {
 // For a chat client
 const chatClient = new AzureOpenAiChatClient({ modelName: 'gpt-4o' });
 // For an embedding client
-const embeddingClient = new AzureOpenAiEmbeddingClient({ modelName: 'gpt-4o' });
+const embeddingClient = new AzureOpenAiEmbeddingClient({
+  modelName: 'text-embedding-3-small'
+});
 ```
 
 In addition to the default parameters of Azure OpenAI and LangChain, additional parameters can be used to help narrow down the search for the desired model:
 
 ```ts
 const chatClient = new AzureOpenAiChatClient({
   modelName: 'gpt-4o',
-  modelVersion: '24-07-2021',
```

Comment:
I believe we need to use the english version of the SAP Note. Otherwise, the translation is sometimes messing up with the date format etc.

Reply:
I feel since we are saying "In addtion to the default parameters", it is not a bad idea to show user some other value possible.

Reply:
And yes, the machine translated version apparently also converts the date format.